libqtile/core/manager.py

```diff
@@ -796,7 +798,10 @@ def _select(self, name: str, sel: Optional[Union[str, int]]) -> Optional[Command
             if sel is None:
                 return self.current_window
             else:
-                return self.windows_map.get(sel)  # type: ignore
+                windows = {
+                    k: v for k, v in self.windows_map.items() if isinstance(v, CommandObject)
+                }
+                return windows.get(sel)  # type: ignore
```

Comment:
Do you know which mypy error is raised here without the comment?

Reply:
Something than can be fixed if we add a type hinting line before line 801...

Reply:
Nice. I fantasise about type annotating the whole codebase so this makes me happy.

Reply:
You should keep that kind of thing to yourself...